st3/sublime_lib/_util/yaml.py

```diff
@@ -0,0 +1,29 @@
+import sublime
+import re
+
+
+__all__ = ['parse_simple_top_level_keys']
+
+
+def parse_simple_top_level_keys(text):
+    return dict(
+        map(_parse_yaml_value, match.groups())
+        for match in re.finditer(r'(?m)^(\S.*?\s*): *(.+) *$', text)
```

Comment:
Not changed in this PR, but we might want to check for comments, although I doubt people are using them in this line

Reply:
Done.